source/base/DateTime.ooc

```diff
@@ -112,7 +112,7 @@ DateTime: cover {
 	//	%ss - second
 	//	%zzzz - millisecond
 	// <param name="format">output format specification</param>
-	toStringFormat: func (format: String) -> String {
+	toString: func ~format (format: String) -> String {
```

Comment:
Because then you don't need 2 overloads, all you need is the one with the default parameter.